reference/docs-conceptual/docfx.json

```diff
@@ -15,14 +15,15 @@
             "ms.tgt_pltfr": "windows, macos, linux",
             "author": "sdwheeler",
             "ms.author": "sewhee",
+            "titleSuffix": "Microsoft Docs",
```

Comment:
In Azure titleSuffix is used for branding, not platform.  Should this be "PowerShell" instead?

Reply:
fixed